deploy-board/deploy_board/templates/environs/envs_landing.html

```diff
@@ -9,6 +9,116 @@
 
 {% load utils %}
 
+{% block side-panel %}
+<div class="panel panel-default">
+    <div class="panel-heading clearfix">
+        <h4 class="panel-title pull-left">Environments</h4>
+    </div>
+    <div class="row">
+        <button id="newEnvBtnId" type="button" class="deployToolTip btn btn-default btn-block"
+            data-toggle="tooltip" title="Create a new deploy environment">
+            <span class="glyphicon glyphicon-file"></span> Create Environment
+        </button>
+    </div>
+    <div class="row">
+        <a href="/envs/deploys" type="button" class="deployToolTip btn btn-default btn-block"
+            data-toggle="tooltip" title="List all the deployments">
+            <span class="fa fa-list-ul"></span> All Deployments
+        </a>
+    </div>
+    {% if envs_tag|isDisabledEnvTag %}
+    <div class="row">
+    <button type="button" class="deployToolTip btn btn-default btn-block"
+            data-toggle="modal" data-target="#enableConfirmModelId"
+            title="Allow environment changes for all environments">
+    <span class="glyphicon glyphicon-ok-circle"></span> Enable All Envs
+    </button>
+    </div>
+    {% else %}
+    <div class="row">
+        <button type="button" class="deployToolTip btn btn-default btn-block"
+                data-toggle="modal" data-target="#disableConfirmModelId"
+                title="Disable new environment changes for all environments
+                to avoid people make any new config and deploy changes">
+        <span class="glyphicon glyphicon-ban-circle"></span> Disable All Envs
+        </button>
+    </div>
+    {% endif %}
+</div>
+<div class="panel panel-default">
+    <div class="panel-heading clearfix">
+        <h4 class="panel-title pull-left">Recent Environments</h4>
+    </div>
+    <div id="recentEnvsId">
+    </div>
+</div>
+
+<!-- Disable all envs dialog -->
+<div class="modal fade" id="disableConfirmModelId" tabindex="-1" role="dialog"
+     aria-labelledby="disableconfirmModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <form id="disableConfigFormId" class="form-horizontal" role="form" method="post" action="/envs/disable/">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
+                <h4 class="modal-title">Disable Confirm</h4>
+            </div>
+            <div class="modal-body">
+                <p>Are you sure to limit new configuration and deploy changes for every environments?</p>
```

Comment:
Are you sure to disable all new deploys and configuration changes for all environments?